codegen/Package.swift

```diff
@@ -103,7 +105,7 @@ if let smithySwiftDir = ProcessInfo.processInfo.environment["SMITHY_SWIFT_CI_DIR
     ]
 } else {
     package.dependencies += [
-        .package(path: "~/Projects/Amplify/SwiftSDK/smithy-swift"),
-        .package(path: "~/Projects/Amplify/SwiftSDK/aws-sdk-swift"),
+        .package(path: "../../smithy-swift"),
+        .package(path: "../../aws-sdk-swift"),
```

Comment:
This should remove the last requirement that the SDK be installed at `~/Projects/Amplify/SwiftSDK` for the project or any of its subprojects to function correctly in development. 🎉

Reply:
(installing `aws-sdk-swift`, `smithy-swift`, and `aws-crt-swift` in the same dir is still required but it can be anywhere on the file system)